docs/hooks/order-select.md

```diff
@@ -0,0 +1,366 @@
+# `order-select`
+
+| Metadata | Value
+| ---- | ----
+| specificationVersion | 1.0
+| hookVersion | 0.1.0
+
+## Workflow
+
+The `order-select` hook fires when a clinician selects one or more orders to place for a patient. 
```

Comment:
I assume this is a generic 'order-select' hook but all examples are for medications. If this applies to labs and procedures, please consider adding here.